docs/src/user/tasks.rst

```diff
@@ -102,74 +102,209 @@ Prediction Format
 
 A array of integers representing the list of predicted tag for each token, in sequence, for the sentence.
 
-TABULAR_CLASSIFICATION
+
+
+QUESTION_ANSWERING
 --------------------------------------------------------------------
 
-Dataset Type
+COVID19 Task Dataset Format
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-:ref:`dataset-type:TABULAR`
+:ref:`dataset-type:QUESTION_ANSWERING`
 
-The following optional train arguments are supported:
 
-    =====================       =====================
-    **Train Argument**          **Description**
-    ---------------------       ---------------------        
-    ``features``                List of feature columns' names as a list of strings (defaults to first ``N-1`` columns in the CSV file)
-    ``target``                  Target column name as a string (defaults to the *last* column in the CSV file)
-    =====================       =====================
+Dataset can be used to finetune the SQuAD pre-trained Bert model. 
 
-The train & validation datasets should have the same columns. 
+- The dataset zips folders containing JSON files. JSON files under different level folders will be automaticly read all together. 
+
+Dataset structure example:
+
+.. code-block:: text
+
+    /DATASET_NAME.zip
+    │
+    ├──FOLDER_NAME_1                                              # first level folder
+    │  └──FOLDER_NAME_2                                           # second level folder, not necessarily to be included
+    │      └──FOLDER_NAME_3                                       # third level folder, not necessarily to be included
+    │           ├── 003d2e515e1aaf06f0052769953e8.json            # JSON file name is a random combination of either alphabets/numbers or both
+    │           ├── 00a407540a8bdd.json
+    │           ...
+    │
+    ├──FOLDER_NAME_4                                              # first level folder
+    │  ├── 0015023cc06b5362d332b3.json
+    │  ├── 001b4a31684c8fc6e2cfbb70304354978317c429.json
+    │  ...
+    ...
+    │
+    └──metadata.csv                                               # if additional information is provided for above JSON files, user can add a metadata.csv
+
+- JSON file includes ``body_text``, providing list of paragraphs in full body which can be used for question answering. ``body_text`` can contain different entries, only the "text" field of each entry will be read. 
+
+1. For JSON files extracted from papers, it comes that one JSON file for one paper. And if additional information is given in metadata.csv for papers, each JSON file and each metadata.csv entries are linked via ``sha`` values of both.
```

Comment:
do i need to do any transformation of the original dataset to follow this format?

Reply:
no transformation needed for original datasets.